geomesa-accumulo/geomesa-accumulo-tools/bin/geomesa

```diff
@@ -57,7 +57,7 @@ fi
 GEOMESA_CP="${GEOMESA_CONF_DIR}:$(findJars $GEOMESA_LIB):${ACCUMULO_CONF_DIR}:${HADOOP_CONF_DIR}"
 
 # Add user defined directories to the classpath
-if [[ -z "$GEOMESA_EXTRA_CLASSPATHS"  ]]; then
+if [[ ! -z ${GEOMESA_EXTRA_CLASSPATHS+x}  ]]; then
```

Comment:
I guess it works but I don't really understand why you wouldn't just do 
`if [[ -n "${GEOMESA_EXTRA_CLASSPATHS}" ]]; then`
which seems more readable.